ISISComputingGroup/EPICS-CCD100
# Ticket 6556: Move emulator and tests to support

See https://github.com/ISISComputingGroup/IBEX/issues/6555

This moves the test and emulator for the CCD100 into this directory.

To test:
* With other branches pulled run `make ioctests` in this directory and confirm that the tests run and pass
* Note that you will not see the output of the tests until they are finished unless you run `set "MAKEFLAGS=-w -j 6"` first
